changelog

```diff
@@ -1,3 +1,96 @@
+      * The weapon jam chance from scrambling is no longer affected by the current energy level. (@Anarchist2)
+      * The ability of a homing projectile to track targets behind it, reduce its acceleration, and lead a target can now be set independently of one another. (@Anarchist2)
+      * Ramscoop fuel and solar energy collection are now based on a ship's actual distances to the stars in a system, instead of the distance to the system center. (@TomGoodIdea)
+      * The autopilot can now account for insufficient crew. (@NobodyIII, @tibetiroka)
+      * Added a preference allowing a user to control if passengers and/or cargo are loaded into the flagship or escorts first. (@TomGoodIdea)
+      * The autopilot AI can now more readily and effectively make use of reverse thrust. (@tibetiroka)
+  * User interface:
+    * The "Show mini-map" preference now has the option to have the mini-map shown at all times. (@Amazinite)
+    * The radar no longer shows invisible projectiles, and projectiles with a blast radius will now appear larger. (@TomGoodIdea)
+    * The landing message will no longer be repeated if the player selects the planet they are already landing on. (@Amazinite)
+    * The map outfitter and shipyard panels now sort by series and index, instead of alphabetically. (@Anarchist2)
+    * The tooltip for the "Toggle turret tracking" control now directs users to the "Turret tracking" setting for more details about what it does. (@xobes)
+    * Refactored and consolidated tooltip handling into its own class and improved tooltip positioning in some cases. (@Amazinite)
+  * Under the hood:
+    * Sound loading code now uses references to shared_ptrs instead of making copies. (@TomGoodIdea)
+    * Consolidated code for listing items into the Format class. (@TomGoodIdea)
+    * Refactored ship order handling to support concurrent orders. (@TomGoodIdea)
+    * Enabled large address awareness on 32-bit Windows builds, allowing the use of more than 2 GB of memory. (@TomGoodIdea)
+    * Refactored audio loading, playback, and effects into separate classes. (@tibetiroka)
+    * Various C++ code style fixes. (@TomGoodIdea, @xobes)
+    * Cleaned up the EsUuid class, and moved the comparator to its own file. (@TomGoodIdea)
+    * Consolidated left and right click handling functions into a single click handling function. (@TomGoodIdea)
+    * Apply a uniform style for template declarations throughout the codebase. (@TomGoodIdea)
+    * Apply a uniform style to conversation labels. (@warp-core)
+    * Added an overload to "FillShader::Fill" that takes the region to fill as a "Rectangle" instead of a pair of "Point"s. (@TomGoodIdea)
+    * Removed unused icon setting code and files. (@TomGoodIdea)
+    * Various improvements and fixes to timers on Windows. (@TomGoodIdea)
+  * Documentation:
+    * Replaced the Windows metadata file description with the application names because it was used as the process name in Task Manager. (@TomGoodIdea)
```

Comment:
```suggestion
    * Replaced the Windows metadata file description with the application name because it was used as the process name in Task Manager. (@TomGoodIdea)
```